### ISSUE TITLE
Conflict with Alternative's <|>

### ISSUE THREAD

**Issue by afcowie** [Daniel-Diaz/matrix#41]
I'm sure you're aware, but an operator named `<|>` is a part of the [Alternative](https://hackage.haskell.org/package/base-4.9.0.0/docs/Control-Applicative.html#t:Alternative) typeclass from Control.Applicative. Since that's in **base** now, I don't think it's wise for **matrix** to export a function with the same name for joining blocks.

Not the end of the world.

**Comment by octopuscabbage** [comment]
Any proposals as to what it's changed to?

**Comment by Daniel-Diaz** [comment]
Right, `<|>` was a terrible name to choose. We need to change it. I'm happy to see your suggestions. Maybe `<||>`? The issue with this is backwards compatibility. We should first make a release with both the old and the new name, with the old being marked as deprecated.

**Comment by octopuscabbage** [comment]
Seems like the best alternative (no pun intended) I did some Googling and couldn't find a better looking operator. Would we change the vertical join?

**Comment by afcowie** [comment]
If two lines is your thing, then `<=>`?

Personally I would just have gone with 

```
one
`above`
two
```
and
```
three `beside` four
```

but that's just me. :)

**Comment by octopuscabbage** [comment]
Yeah, the fact that we can't really come up with a good operator makes me think that concatenation isn't really worth an operator. I think your suggestion of above and beside is fine, except it looks kind of weird not infix. Maybe verticalConcat and horizontalConcat or something.
